testbed/lib/main.dart

```diff
@@ -226,13 +226,14 @@ class _MyHomePage extends StatelessWidget {
       appBar: AppBar(
         title: Text(title),
         actions: <Widget>[
-          new IconButton(
-            icon: new Icon(Icons.color_lens),
-            tooltip: 'Change theme color',
-            onPressed: () {
-              _changePrimaryThemeColor(context);
-            },
-          ),
+          if (!Platform.isWindows)
```

Comment:
Can you explain what problem you are trying to solve here? Per [the README](https://github.com/google/flutter-desktop-embedding/blob/master/testbed/README.md) `testbed` is intended for use by people working on FDE. Hiding the unimplemented buttons just adds a step for anyone working on adding Windows support, so I don't see the benefit.

Reply:
Ok instead of Hiding i will try to implement them when i got time